sputnikdao2/tests/test_general.rs

```diff
@@ -242,3 +242,66 @@ fn test_failures() {
         None,
     ));
 }
+
+/// Test payments that fail
+#[test]
+fn test_payment_failures() {
+    let (root, dao) = setup_dao();
+    let user1 = root.create_user(user(1), to_yocto("1000"));
+    let whale = root.create_user(user(2), to_yocto("1000"));
+
+    // Add user1
+    add_member_proposal(&root, &dao, user1.account_id.clone()).assert_success();
+    vote(vec![&root], &dao, 0);
+
+    // Set up fungible tokens and give 5 to the dao
+    let test_token = setup_test_token(&root);
+    call!(
+        dao.user_account,
+        test_token.mint(to_va(dao.user_account.account_id.clone()), U128(5))
+    )
+    .assert_success();
+    call!(
+        user1,
+        test_token.storage_deposit(Some(to_va(user1.account_id.clone())),Some(true)),
+        deposit = to_yocto("125")
```

Comment:
Not so important, but this should be `to_yocto("0.00125")`. This is the cost for creating an account for a fungible token (assuming the prefix is only one letter).

If it is `to_yocto("125")`, the rest will be refunded back, so there's no harm, but still. I think the test would be more precise with `to_yocto("0.00125")`.